services/statusaccounts/path_decoder.go

```diff
@@ -0,0 +1,224 @@
+package statusaccounts
```

Comment:
I don't see anything wrong with that. It would be `github.com/status-im/keycard-go/derivationpath` so looks fine. However, `keycard-go` should be tagged properly and watched for breaking changes.